sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1311,6 +1311,10 @@ export class ApplicationService extends RecordDataModelService<Application> {
         id: true,
         applicationNumber: true,
         isArchived: true,
+        data: {
+          studystartDate: true,
```

Comment:
We use the dynamic data from the application for some specific situations. In general, we will not be consuming, for instance, `studystartDate` and `selectedOfferingDate` from the dynamic data.
Relevant data, like offering data, should be stored somewhere else in DB and should be consumed from there 😉